source/query-bar.txt

```diff
@@ -306,49 +306,23 @@ as favorites. To view the list of queries saved as favorites, click on
 .. figure:: /images/compass/query-history-favorite.png
    :figwidth: 316px
 
-.. class:: hidden
-
-   .. toctree::
-      :titlesonly:
-
-      /export-query-to-language
-
 Limitations
 -----------
 
-Limitations apply to the following query operators:
-
-- :manual:`$near
+- The query bar fields do not support :manual:`$near
```

Comment:
Nice! One tiny nit is to replace `fields` with `options` here so we are consistent between the bullets. (I know `fields` was there before though 😃)

Reply:
Done.